api/apps/api/src/modules/scenarios/scenarios.service.ts

```diff
@@ -947,4 +957,27 @@ export class ScenariosService {
     }
     return right(true);
   }
+
+  async releaseLock(
+    scenarioId: string,
+    userId: string,
+  ): Promise<
+    Either<
+      typeof forbiddenError | typeof lockedByAnotherUser | typeof noLockInPlace,
+      void
+    >
+  > {
+    const scenario = await this.getById(scenarioId, {
+      authenticatedUser: { id: userId },
+    });
+    if (isLeft(scenario)) {
+      return left(forbiddenError);
+    }
```

Comment:
I am not too sure about this: `scenario`'s type is `Either<GetScenarioFailure | typeof forbiddenError, Scenario>`, but here we're compressing the possible left values to `forbiddenError` - I'd pass the upstream `left` through here.